rero_ils/modules/items/api/circulation.py

```diff
@@ -477,23 +477,30 @@ def cancel_item_request(self, pid, **kwargs):
             item = self
```

Comment:
Maybe we can move `item = self` before the `if`'s end delete it everywhere else.